rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -95,22 +95,28 @@
         <dependency>
             <groupId>org.jboss.narayana.rts</groupId>
             <artifactId>narayana-lra</artifactId>
+            <version>${project.version}</version></dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-jackson</artifactId>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.jboss.narayana.rts</groupId>            
+            <artifactId>narayana-lra</artifactId>
```

Comment:
This adds narayana-lra dependency a second time (from the one around line 97)

Reply:
Thanks for the review @tomjenkinson . I remove it.